l2geth/cmd/geth/main.go

```diff
@@ -153,6 +153,7 @@ var (
 		utils.Eth1SyncServiceEnable,
 		utils.Eth1CanonicalTransactionChainDeployHeightFlag,
 		utils.Eth1L1CrossDomainMessengerAddressFlag,
+		utils.Eth1L1FeeWalletAddressFlag,
```

Comment:
Nit: unified names between geth and contracts

Reply:
The name of the variable is l1FeeWallet

Reply:
The name of the contract is `OVM_SequencerFeeVault` which is different than `L1FeeWallet`

Reply:
Ah I see now, its the same as the name of the variable in the contract